### ISSUE TITLE
Get Mailing List custom fields in the Campaign template

### ISSUE THREAD

**Issue by scandella** [putyourlightson/craft-campaign#250]
### Question
I need to output some custom Mailing List fields into the Campaign template (like some legal informations attached to certain Mailing List, for instance).
I was able to get all the Mailing List's ID of the Sendout element within the Campaign template, but how can I get the specific Mailing List ID which is in use when the Campaign is sent in order to display a custom field of this particular Maling List?
Like the way we can access some custom Contact fields in the Campaign Template via the `contact` element. 

Best,

**Comment by bencroker** [comment]
There currently isn't any way to reliably get the mailing list, since a contact can be subscribed to multiple mailing lists that the sendout is being sent to. What you could do is find which mailing list(s) the sendout and contact have in common.

```twig
{% set commonMailingLists = sendout.mailingLists|filter(mailingList => mailingLists in contact.mailingLists) %}
{% set mailingList = commonMailingLists|first %}
```

**Comment by bencroker** [comment]
I'm going to change this issue to a feature request to make `mailingList` available as a variable in the template, since it seems like a useful thing to have.

**Comment by bencroker** [comment]
Added in https://github.com/putyourlightson/craft-campaign/commit/48cbd9e39ab809327e87e147bdd8f083b7753e0b for the next release.

**Comment by bencroker** [comment]
Released in version 1.21.0.
